recipes/pymagewell/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set owner = "KCL-BMEIS" %}
+{% set name = "pymagewell" %}
+{% set version = "0.1.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz  
+    sha256: bc906d1c5c70146d4131ff8932d969279c2a97f3f8d8792ba6093b2b1bd972c9
+  - url: https://github.com/{{ owner }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 4a136f5972d6e35fa01663a4e38ea4811b2d5918c03f0e9254ddb41d41a14fd2
+    folder: test-data
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<39]
+  skip: True  # [linux]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy >=1.21.4
+    - pywin32 >=303  # [win]
+    - python
+    - libopencv >=4.5.5
+    - opencv >=4.5.5
+    - py-opencv >=4.5.5
+test:
+  imports:
+    - mwcapture
+    - pymagewell
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - test-data/tests
+  commands:
+    - pip check
+    - pytest  # [win]
```

This is already a Windows only pkg, no need for the pre-processing selector.
```suggestion
    - pytest  # [win]
```

```suggestion
    - pytest
```